src/cc65/main.c

```diff
@@ -951,6 +962,9 @@ int main (int argc, char* argv[])
 
                 case 'O':
                     IS_Set (&Optimize, 1);
+                    IS_Set (&CodeSizeFactor, 100);
+                    IS_Set (&EnableRegVars, 0);
+                    IS_Set (&InlineStdFuncs, 0);
```

Comment:
With all options in gcc AFAIK, if conflicting, the last one succeeds. Yes, I was referring to the -O options in gcc (-O<num>, -Ofast, -Os, -Og).
But, as I said, I think the current cc65 behaviour should be retained.